spec/controllers/api/tasks_controller_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
+
+RSpec.describe Api::TasksController, type: :controller do
+  render_views
+
+  let!(:user) { create(:user) }
+  let!(:workspace) { create(:workspace, user: user) }
+  let!(:project) { create(:project, workspace: workspace) }
+  let!(:task1) { create(:task, project: project) }
+  let!(:task2) { create(:task, project: project) }
+  let!(:task_valid_params) { { title: "test_task"} }
+  let!(:task_invalid_params) { { title: nil} }
+
+  before do
+    sign_in user
+  end
+
+  describe 'GET /projects/:project_id/task/:id' do
+    it 'should show task page' do
+      get :show, as: :json, params: { project_id: project.id, id: task1.id }
+      parsed_response = JSON.parse(response.body)
+      expect(response.body).to include(task1.title)
+      expect(response).to have_http_status(200)
+    end
+  end
+
+  describe 'POST /projects/:project_id/tasks' do
+    context "with valid attributes" do
+      it 'should create task' do
+        post :create, as: :json, params: {
+          project_id: project.id,
+          task: {
+            title: Faker::Lorem.sentence,
+            description: Faker::Lorem.paragraph
+          }
+        }
+
+        expect{ post :create, params: { project_id: project.id, task: { title: Faker::Lorem.sentence, description: Faker::Lorem.paragraph }}}.to change(Task, :count).by(1)
```

Comment:
very long line :)
bellow you have params written in few lines, please save consistency

Reply:
oooo it is the same params then above :)